addons/missileGuidanceCompat/CfgAmmo.hpp

```diff
@@ -4,6 +4,11 @@ class CfgAmmo {
     class M_Titan_AT;
     class Bo_GBU12_LGB;
 
+    class ShellBase;
+    class ace_dragon_serviceCharge: ShellBase {
+        soundSetExplosion[] = {};
```

Comment:
I don't know much about ARMA audio, but make sure that a player can hear the dragon flying towards them. I'll accept after it's tested

Reply:
JSRS adds it's sound sets as it's an easy way to override / customize sound. The soundset would normally be undefined, but with JSRS inserts an entry. With JSRS's entry, at a distance you hear JSRS medium-sized explosions instead of the actual service charge pops which is confusing to say the least. Again, video when I get off work

Reply:
First half is w/o the fix, second half is with the fix
https://github.com/user-attachments/assets/75555729-a1a2-4825-806f-2c6e703c4872

Reply:
Some people will miss the soundscape; but I am fine with it either way. I'm resolving it for now, I was just worried that the audio would be completely messy

Reply:
I can edit together a video that's clearer on the changes and submit a PR if it should.